scripts/git-clone-all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+REPOSITORIES=(DNC-DShop DNC-DShop.Api DNC-DShop.Common DNC-DShop.Messages DNC-DShop.Services.Customers DNC-DShop.Services.Identity DNC-DShop.Services.Notifications DNC-DShop.Services.Operations DNC-DShop.Services.Orders DNC-DShop.Services.Products DNC-DShop.Services.Signalr DNC-DShop.Services.Storage DNC-DShop.Monolith DNC-DShop.Web)
+cd ../..
```

Comment:
I'm not quite sure about this. All scripts were supposed to be kept in root folder (same level as each repo) thus I'd remove this line.

Reply:
Agreed, on the other hand - useful script, I had something similar on my local drive, yet forgot to include it in this repo, thanks :).